src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -368,31 +369,31 @@ private int ParseAuditLevel()
 
             if (auditLevel == null)
             {
-                return 1;
+                return 0;
             }
 
             if (string.Equals("low", auditLevel, StringComparison.OrdinalIgnoreCase))
```

Comment:
>low [](http://example.com/codeflow?start=31&length=3)

I'm a bit worried that these are not constants defined somewhere. A typo in a later change could make the strings to be mismatched when written/read from the properties.